src/main/java/se/fylling/jarl/UndefinedException.java

```diff
@@ -0,0 +1,9 @@
+package se.fylling.jarl;
+
+public class UndefinedException extends Exception {
```

Comment:
We should wire this up in the evaluator. But I can do that as a second step if you want to keep going on the built-in side of the equation.

Reply:
Please do! :)